src/redux/features/editProfileSlice.ts

```diff
@@ -108,6 +117,7 @@ const updateStateFromPayload = (
   state.videoLink = videoLink
   state.categories = mainSubjects
   state.professionalBlock = professionalBlock || initialProfessoinalBlock
+  state.aboutStudent = aboutStudent || initialAboutStudent
```

Comment:
I think that nullish coalescing will be more meaningful here (as default value), but I see, that other developers used logical OR here and it's seems a little confusing. 
So, it's up to you

```suggestion
  state.aboutStudent = aboutStudent ?? initialAboutStudent
```

Reply:
by default, `aboutStudent` field is not null, it's an object with properties that contain empty strings. so it would be a mistake to use `??` here